desc/objectives/_stability.py

```diff
@@ -367,3 +369,278 @@ def compute(self, params, constants=None):
             profiles=constants["profiles"],
         )
         return constants["transforms"]["grid"].compress(data["magnetic well"])
+
+
+class BallooningStability(_Objective):
```

Comment:
Is there any chance we ever add a finite-N ballooning solver? If so, we should change this name to something like `InfiniteNBallooningStability` to clarify.

Reply:
There is no such thing as a finite-N ballooning equation.